DEVELOPMENT.md

```diff
@@ -270,7 +270,13 @@ To look at the logs for individual `TaskRuns` or `PipelineRuns`, see
 If you need to add a new CRD type, you will need to add:
 
 1. A yaml definition in [config/](./config)
-1. Add the type to the cluster roles in
-   [200-clusterrole.yaml](./config/200-clusterrole.yaml)
+1. Add the type to the cluster roles in:
+   - [200-clusterrole.yaml](./config/200-clusterrole.yaml)
+   - [clusterrole-aggregate-edit.yaml](./config/clusterrole-aggregate-edit.yaml)
+   - [clusterrole-aggregate-view.yaml](./config/clusterrole-aggregate-view.yaml)
+1. Add go structs for the types in [pkg/apis/pipelines/v1alpha1](./pkg/apis/pipeline/v1alpha1)
```

Comment:
NIT: I'm not sure if we should include this, or only this. I guess the API types are the bare minimum, but perhaps we should include the [type]_validation and [type]_default modules as well; and perhaps the controller as well.

Reply:
hmmm, so in order to register with the webhook, the types have to be both `Defaultable` as well as `Validatable`. But probably worth calling this out explicitly.